packages/web/src/app/onboard/page.tsx

```
@@ -1,28 +1,379 @@
+    icon?: React.ReactNode
+}
 
-export default async function Onboarding() {
+export default async function Onboarding({ searchParams }: OnboardingProps) {
+    const providers = getAuthProviders();
+    const org = await getOrgFromDomain(SINGLE_TENANT_ORG_DOMAIN);
     const session = await auth();
-    if (!session) {
-        redirect("/login");
+
+    if (org && org.isOnboarded) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)